src/common/numa.cc

```diff
@@ -156,6 +156,32 @@ static int easy_readdir(const std::string& dir, std::set<std::string> *out)
   return 0;
 }
 
+std::string get_task_comm(pid_t tid)
```

Comment:
nit, make this method a static method or put it in an anonymous namespace, as it is not used elsewhere.